src/Distances.jl

```diff
@@ -28,32 +29,24 @@ export
     Minkowski,
     Jaccard,
     BrayCurtis,
-    RogersTanimoto,
-
-    Hamming,
+    RogersTanimoto, Hamming,
```

Comment:
Intended change?

Reply:
No, it must have been the autoformat, undoing